### PR TITLE
[format-thousands] add types to format-thousands

### PR DESCRIPTION
**Pull request by c1495616js**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/format-thousands/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Type definitions for format-thousands 2.0
+// Project: https://github.com/VovanR/format-thousands
+// Definitions by: Jerry Wang <https://github.com/c1495616js>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = format_thousands;
+
+declare function format_thousands(number?: number | string, option?: string | { separator?: string; formatFourDigits?: boolean }): string;
```
Comment (peterblazejewicz) [inline types/format-thousands/index.d.ts:8]:
```suggestion
export = formatThousands;

declare function formatThousands(number?: number | string, option?: string | { separator?: string; formatFourDigits?: boolean }): string;
```
it's named `formatThousands` (just to stick to the JS convention for naming things):
https://github.com/VovanR/format-thousands#usage

Comment (c1495616js) [reply types/format-thousands/index.d.ts]: Thanks @peterblazejewicz for the review.
Addressed it.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@c1495616js thanks!

**Comment by c1495616js** [comment]
Ready to merge

**Pull request merged**